po/it.po

```diff
@@ -0,0 +1,36 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the inputmethod-indicator package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: inputmethod-indicator\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-11-03 06:39+0000\n"
+"PO-Revision-Date: 2020-11-03 08:44+0000\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Fabio Zaramella\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.3\n"
+
+#: src/Indicator.vala:76
+msgid "IBus Daemon is Not Running"
+msgstr "Il demone IBus non è in esecuzione"
+
+#: src/Indicator.vala:77
+msgid "Click \"Input Method Settings…\" to show available input method engines."
+msgstr "Fai clic su \"Impostazioni metodi di inserimento\" per visualizzare i metodi disponibili."
```

Comment:
```suggestion
msgstr "Fai clic su \"Impostazioni metodi di inserimento…\" per visualizzare i metodi disponibili."
```

I don't understand Italian, but don't you need the trailing ellipsis as the original sentence and the translation in the line 30 have?

Reply:
Of course, thanks for noticing! 😅